### PR TITLE
Update activate_partynpc.tpa

### PR DESCRIPTION
**Pull request by Selphira**
Fix this problem
```
OR(6)
  OR(6)
  IsOverMe(Player1)
  IsOverMe(Player2)
  IsOverMe(Player3)
  IsOverMe(Player4)
  IsOverMe(Player5)
  IsOverMe(Player6)
  IsOverMe(Player2)
IsOverMe(Player3)
IsOverMe(Player4)
IsOverMe(Player5)
IsOverMe(Player6)
```

### CODE REVIEW

**Comment by Gitjas** [comment]
Did you encounter this ina modded game? Please tell me which game and post your weidu.log. Thanks!

**Comment by Gitjas** [comment]
Because this is the bdrom03.bcs of the unmoddd game:
```
IF
	Global("bd_viconia_romancecheck","global",1)
	Global("bd_ViconiaRomance1","GLOBAL",0)
	InParty("Viconia")  // Viconia
	IsOverMe(Player1)
THEN
	RESPONSE #100
		SetGlobal("bd_viconia_romanceactive","global",1)
		SetGlobal("bd_ViconiaRomance1","GLOBAL",1)
END

IF
	Global("bd_viconia_romanceactive","global",1)
	Global("bd_ViconiaRomance1","GLOBAL",1)
	InParty("Viconia")  // Viconia
	TriggerOverride("Viconia",See(Player1))
	IfValidForPartyDialog("Viconia")  // Viconia
	!ActuallyInCombat()
THEN
	RESPONSE #100
		ActionOverride("Viconia",StartDialogueNoSet(Player1))
END
```
As you can see, there is only "IsOverMe(Player1)" so the mod would patch correctly. There is two possibilities I can think of right away for what you encountered:
1. another mod changes the trigger the same way like J's SoD Tweak Pack does
2. for some reason, the tweak pack was installed twice in your game (which can happen if a mod is not deinstalled properly but the install info was removed.)

In short: this is not a bug in the mod and accepting your PR would break the mod's intention because the script would only check for Player1.

**Comment by Selphira** [comment]
Hello, here is the changelog for this file on my installation

```
Mods affecting BDROM03.BCS:
00000: /* created or unbiffed */ ~EET/EET.TP2~ 3 0 // EET core (importation de ressource)V13.4
00001:  ~C#SODTWEAKS/SETUP-C#SODTWEAKS.TP2~ 3 2 // Ne manquez pas de dialogues entre NPC-PCPour les PNJ du groupe uniquementv11.1
00002:  ~CDTWEAKS/SETUP-CDTWEAKS.TP2~ 2 1010 // Davantage d'interjectionsv16
00003:  ~STUTTERDEBUG/STUTTERDEBUG.TP2~ 2 1 // Stutter Debug ToolPatching de tous les scripts a l'exception des blocs NoAction() - choisissez cette option lors de la premiere installation (l'installation peut prendre un certain temps !)4.0

```

**Comment by Gitjas** [comment]
Thank you very much! I'll look into this when I'll find the time.

**Comment by Selphira** [comment]
I don't know if you've noticed, but the code I'm deleting is duplicate code.

The bdrom03.bcs file is modified 2 times (on line 31 and line 44)

Is this code behaviour intentional?

**Comment by Gitjas** [comment]
I was notified about me being hasty elsewhere, too. - You are all so much smarter than me! No, duplicating this is nonsense and would lead to the bug, obviously. Duh.
Thank you again for reporting and being persistent!
